File: nodes/mc/gomc/js/nodes/excite/excite.js
MNode = require('libmango')

function Excite(){
    this.node = new MNode();
    this.node.iface.add_handlers({'excite':this.excite});
    this.node.run();
}

Excite.prototype.excite = function(header,args){
    return ["excited",{'message':args['message']+'!'}]
}

new Excite();

File: nodes/mc/gomc/py/nodes/excite/excite.py
from libmango import *

class excite(m_node):
    def __init__(self):
        super().__init__(debug=True)
        self.interface.add_interface({'excite':self.excite})
        self.debug_print("running")
        self.run()
    def excite(self,header,args):
        return "excited",{'message':args['message']+'!'}
    
excite()
